src/main/java/com/exasol/projectkeeper/ProjectKeeperVerifyMojo.java

```diff
@@ -21,27 +20,28 @@
     private static final String INVALID_STRUCTURE_MESSAGE = "This projects structure does not conform with the template.";
 
     @Override
-    public void execute() throws MojoExecutionException, MojoFailureException {
-        final var hadFindingsWithFix = new AtomicBoolean(false);
-        final var hadFindingsWithoutFix = new AtomicBoolean(false);
-        getValidators().forEach(validator -> validator.validate(finding -> {
-            getLog().error(finding.getMessage());
-            if (finding.hasFix()) {
-                hadFindingsWithFix.set(true);
-            } else {
-                hadFindingsWithoutFix.set(true);
-            }
-        }));
-        if (hadFindingsWithoutFix.get() && hadFindingsWithFix.get()) {
+    public void execute() throws MojoFailureException {
+        final var findings = getValidators().stream().flatMap(validator -> validator.validate().stream())
+                .collect(Collectors.toList());
+        final var log = getLog();
```

Comment:
To tell the truth, var makes the code less readable here. I have to spend time figuring out the types